modules/admin/proc-enabling-telemetry-using-operator.adoc

```diff
@@ -0,0 +1,41 @@
+[id="proc-enabling-telemetry-using-operator_{context}"]
+= Enabling telemetry using the Operator
+
+You can enable the telemetry data collection feature by using the Operator.
+
+.Procedure
+
+. Perform one of the following steps:
++
+* If you have created the `dynamic-plugins-rhdh` ConfigMap file, add the `analytics-provider-segment` plugin to the list of plugins and set its `plugins.disabled` parameter to `false`.
```

Comment:
Same as for Helm. If telemetry is disabled, that means that the configuration is already there and he just needs to find the `./dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment` and set it to `disabled: false`.